ros/lib/rbac_interface.py

```diff
@@ -118,3 +120,79 @@ def _is_mgmt_url(path):
     Small helper to test if URL is for management API.
     """
     return path.startswith("/mgmt/")
+
+
+def set_host_groups(rbac_response):
+    """
+    We now also have to store the host group information we get from inventory.
```

Comment:
@upadhyeammit based on my understanding the text should be `store the user group information we get from RBAC?` correct me if I am wrong 🙌🏻

Reply:
Yeah the next line says that exactly:
~~~
We now also have to store the host group information we get from inventory.
 This comes in the resource definition within the RBAC response:
~~~